wp-content/themes/hurumap/functions.php

```diff
@@ -174,3 +174,14 @@ function register_acf_block_types()
 if (function_exists('acf_register_block_type')) {
     add_action('acf/init', 'register_acf_block_types');
 }
+
+function attachment_search( $query ) {
```

Comment:
Forgot to ask @KhadijaMahanga, why is this method called `attachment_search` when now we're searching everything? attachments, posts, pages, custom post types, etc.